src/GRA.Web/Areas/MissionControl/Views/PerformerScheduling/Performer.cshtml

```diff
@@ -119,25 +127,38 @@
                 </div>
             }
         </div>
-        <div class="form-group">
-            <label>Programs available for the age groups</label>
-            <ul>
-                @foreach (var ageGroup in Model.AgeGroups)
-                {
-                    <li>@ageGroup.Name</li>
-                }
-            </ul>
-        </div>
-        <div class="form-group">
-            <label>Locations</label>
-            <div>
-                Able to perform at
-                <strong>@(Model.Performer.AllBranches || Model.BranchAvailability.Count == Model.System.Branches.Count ? "all" : $"{Model.BranchAvailability.Count} of {Model.System.Branches.Count}")</strong>
-                libraries in your system.
+        @if (!string.IsNullOrEmpty(Model.ReferencesPath))
+        {
+            <div class="form-group">
+                <label>Vendor ID</label>
+                <div>@Model.Performer.VendorId</div>
+            </div>
+            <div class="form-group">
+                <label>Has Clearance?</label>
+                <div>
+                    @(Model.Performer.IsApproved ? "Yes" : "No")
+                </div>
+            </div>
```

Comment:
Text should be "Has Fingerprint Clearance Card" and it should be checking the model property HasFingerprintCard